assets/css/default-calendar-grid.css

```diff
@@ -543,3 +543,22 @@
 .simcal-event-bubble .simcal-event-details .simcal-event-description {
 	margin: 0 0 1em;
 }
+
+.print-calendar-button {
+	margin-top: 20px;
+}
+
+@media print {
```

Comment:
Can we add a class in the top of the page and then apply every CSS inside that? That will ensure this CSS will only apply when print button is clicked.

Reply:
Done